README.md

```diff
@@ -1,2 +1,58 @@
-# morse-code
-Simple ruby exercise
+
+![](https://img.shields.io/static/v1?label=BY&message=JohnFTitor&color=blue)
+![](https://img.shields.io/static/v1?label=BY&message=Reemoz&color=pink)
+
+## Morse code message
+
+> Simple decoder for Morse messages.
+
+
+### Requirements
+- Ruby install 
+
+## Built With
+
+- Languages: _**Ruby**_
+- Frameworks: _**N/A**_
+- Technologies used: _**GIT, GITHUB, LINTERS**_
+
+## Getting Started
+
+### Clone this repository
+
+```bash
+$ git clone git@github.com:JohnFTitor/morse-code.git
+$ cd morse-code
+
+$ bundle install
+```
```

Comment:
[OPTIONAL]

- Here I would recommend including the command `ruby morse_decode.rb` to run the application. Kindly do that before merging.